contracts/test/LONStaking.t.sol

```diff
@@ -332,6 +367,9 @@ contract LONStakingTest is Test {
      *********************************/
 
     function _stake(address staker, uint256 stakeAmount) internal {
+        if (lon.balanceOf(staker) < stakeAmount) {
+            lon.mint(staker, stakeAmount);
+        }
```

Comment:
I feel this is a little implicit, maybe we could derive another function `mintToStake`, `mustStake` or other good namings.

```solidity
function mustStake(staker, stakeAmount) {
    if (lon.balanceOf(staker) < stakeAmount) {
        lon.mint(staker, stakeAmount);
    }
    _stake(staker, stakeAmount)
}
```

Reply:
I ended up moving the `mint` outside out of `_stake` and make it explicit in the tests because only fuzzing tests will need to mint so I think maybe it's not worthy to apply mint logic to all stake helper functions.